sc-tools/sc-config/src/sc_config.cpp

```diff
@@ -66,14 +67,48 @@ std::string ScConfigGroup::operator[](std::string const & key) const
   if (it == m_pathKeys.cend())
     return value;
 
-  if (value[0] == '/')
-    stream << value;
-  else
-    stream << m_configPath << value;
+  std::string valueStr(value);
+  std::string::size_type start = 0;
+
+  while (true)
+  {
+    std::string::size_type end = valueStr.find(ScConfig::PATHS_SEPARATOR, start);
+    std::string path = valueStr.substr(start, end - start);
+
+    path.erase(std::remove_if(path.begin(), path.end(), ::isspace), path.end());
```

Comment:
I cloned sc-machine into folder with name "sc machine" and got warning
![image](https://github.com/user-attachments/assets/247327ed-d952-4c52-8c9d-dbd68f2b192c)